edx_proctoring/statuses.py

```diff
@@ -252,6 +252,12 @@ class InstructorDashboardOnboardingAttemptStatus:
     verified = 'verified'
     error = 'error'
 
+    # The following statuses are not true attempt statuses, but are used when the
+    # user's onboarding profile is approved in a different course.
+    other_course_approved = 'other_course_approved'
+    # Onboarding attempts expire after two years.
+    expiring_soon = 'expiring_soon'
```

Comment:
I am wondering if this should really be a status and instead some attribute we return on the API call for the learner panel specifically. Maybe a `notify_of_pending_expiration` bool or something? I'm worried about overloading the use of status with things that aren't actually a real value for status.  For example I don't think we wanna show this on the instructor side since it's not reflected in Verificient's dashboard that way.

Reply:
I think that makes sense. I meant to ask if we wanted to include the expiring status on the instructor dashboard as well; I'll leave it out in this case.

Reply:
another solution is maybe we just want to have the UI do the date logic here and just return `onboarding_expiration_date`